src/components/molecule/diningCard/DiningCard.tsx

```diff
@@ -0,0 +1,112 @@
+import TextBox from '@/components/atom/TextBox';
+import assets from '@/variables/images';
+import { cva } from 'class-variance-authority';
+import Image from 'next/image';
+import React from 'react';
+
+interface DiningCardProps {
+  type: 'on-site' | 'off-site';
+  img?: string;
+  alt?: string;
+  title?: string;
+  subTitle?: string;
+  content?: string;
+  contact?: string;
+}
+
+const imageWrapper = cva(['min-h-[255px]', 'relative']);
+const flexCol = cva(['flex', 'flex-col', 'items-center']);
+const gapVariants = cva(flexCol(), {
+  variants: {
+    gap: {
+      small: 'gap-1',
+      medium: 'gap-[9px]',
+      large: 'gap-6',
+    },
+  },
+});
+const borderVariants = cva('border border-solid', {
+  variants: {
+    border: {
+      gold: 'w-[51px] border-primary-gold',
+      gray: 'w-[234px] border-[0.5px] border-transparent border-t-primary-gray/50',
```

Comment:
```suggestion
      gray: 'w-[234px] border-[0.5px]  border-t-primary-gray/50',
```
Tailwind에서는 `border` 를 따로 지정하지 않으면 기본적으로 테두리가 없기 때문에,
`border-transparent` 를 추가하지 않아도 `border-t-primary-gray/50` 만으로 같은 효과를 낼 수 있을 것 같습니다!

Reply:
공통 클래스에 border가 들어가있어서 알려주신 코드만으론 수정이 되지 않습니다. 민서님이 말씀하신 거처럼 하려면 border를 공통 클래스에서 내리고  `border-t-[0.5px]`까지 추가해야 합니다. 
```
const borderVariants = cva('border-solid', {
  variants: {
    border: {
      gold: 'w-[51px] border border-primary-gold',
      gray: 'w-[234px] border-t-[0.5px] border-t-primary-gray/50',
    },
  },
});
```

Reply:
공통 클래스까진 고려를 못했네요 알겠습니다! 수고하셨어요~~

Reply:
넹, 근데 말씀하신대로 고치는게 더 보기 깔끔해서 그렇게 수정했어요. 감사함다